src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsbookavideolinkapi/service/events/InboundEventsListener.kt

```diff
@@ -28,8 +28,6 @@ class InboundEventsListener(
   @SqsListener("bvls", factory = "hmppsQueueContainerFactoryProxy")
   @WithSpan(value = "farsight-devs-book_a_video_link_queue", kind = SpanKind.SERVER)
   fun onMessage(rawMessage: String) {
-    log.info("LISTENER: raw message $rawMessage")
```

Comment:
Taking out.  This will start to get quite noisy when we start listening for prisoner release events.